src/odemis/driver/test/xt_client_test.py

```diff
@@ -281,6 +283,158 @@ def test_set_beam_shift(self):
         self.scanner.beamShift.value = (new_beam_shift_x, new_beam_shift_y)
         self.assertAlmostEqual((new_beam_shift_x, new_beam_shift_y), self.scanner.beamShift.value)
 
+    @unittest.skip("Currently this functionality is not supported by the XT client and the code is commented out")
+    def test_apply_autostigmator(self):
+        """
+        Test for the auto functionality of the apply_autostigmator.
+        Runs the apply_autostigmator multiple times and checks (using "get_stigmator()) if the found stigmator values
+        are equal each run. Checks the returned state, the return object and if an error is raised for an incorrect input state.
+        """
+        detector = 'se-detector'  # Test autostigmator only for se-detector, channel=electron1
```

Comment:
Why do you not use the detector from `detector2ChannelName` same way you do for auto focusing?

Reply:
Because per definition this functionality will never be available on other detectors. In theory if other detectors are added, those could be present on other detectors.

Reply:
Ok makes sense and it is important info. Maybe make your comment even more explicit saying that autostigmation only works with the se-detector.